Library/Formula/eg.rb

```diff
@@ -0,0 +1,28 @@
+class Eg < Formula
+  homepage "https://github.com/srsudar/eg"
+  url "https://github.com/srsudar/eg/archive/v0.0.3.tar.gz"
+  sha256 "c257e334f2a2e494f9b9791b92c1d3f7b00da3e2e7ec47f02d720342b466f968"
+
+  def install
+    share.install "eg/examples"
+    inreplace "eg/eg_config.py",
+              /DEFAULT_EXAMPLES_DIR = .+$/,
+              "DEFAULT_EXAMPLES_DIR = os.path.join('#{share}', 'examples')"
+
+    mv "eg", "commands"
+    prefix.install "commands"
```

Comment:
You can cut the above line and make this line `prefix.install "eg" => "commands"` I believe.